src/components/Todo/Todo.module.css

```diff
@@ -0,0 +1,9 @@
+.content {
+	width: 600px;
+	background-color: white;
+	font-family: 'Georgia';
+	padding: 10px;
+	border: hidden;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)